mcap_vendor/CMakeLists.txt

```diff
@@ -35,48 +34,37 @@ macro(build_mcap_vendor)
   file(GLOB _lz4_srcs
     ${lz4_SOURCE_DIR}/lib/*.c)
 
-  add_library(
-    mcap SHARED
+  add_library(${PROJECT_NAME} SHARED
```

Comment:
So I really don't think you want to change the name of the library like this.

That is, the idea behind vendor packages is that they are temporary (for some definition of temporary) stand-ins for the package in question.  In some day in the future, we should be able to remove the vendor package and just get the library from the underlying OS.

Thus, all of our vendor packages actually build and export the actual library name, not a "vendored" version of it.  Someday if `libmcap` is available in Ubuntu, then we can make this package just skip building on Ubuntu.

For those reasons, I do think you should switch back to the `mcap` library and target name.

Reply:
@clalancette Ok, fair point about `libmcap`. I have reverted renames.